src/main/java/bimo/Bimo.java

```diff
@@ -0,0 +1,62 @@
+package bimo;
+import bimo.command.Command;
+
+/**
+ * Represents a chatbot class.
+ */
+public class Bimo {
+    public static final String NAME = "bimo";
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Returns an instance of a chatbot.
+     *
+     * @param filePath File path of file containing list of tasks.
+     */
+    public Bimo(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.tasks = new TaskList(storage.loadFile());
+        } catch (BimoException e) {
+            tasks = new TaskList();
+        }
+    }
```

Comment:
You could add one more break line here, so that there is spacing between the two methods.